package.json

```diff
@@ -50,6 +50,7 @@
     "build": "NODE_ENV=production bash -c 'gulp build && webpack'"
   },
   "dependencies": {
+    "@types/vex-js": "^2.3.29",
```

Comment:
remove it since we dont use it. `yarn delete` oterwise you skip changes to yarn.lock